src/sections/Home/index.scss

```diff
@@ -0,0 +1,30 @@
+.test-block {
+  display: flex;
+  justify-content: space-around;
+  padding: 10px;
+
+  .test-block__list {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex-direction: column;
+    border: 2px solid;
+    padding: 10px;
+    width: 20%;
+
+    .test-block__link {
```

Comment:
syntax should be refactored to `&__link`, same applies to the rest

Reply:
https://github.com/igorizviekov/mt-booking_app/pull/20/commits/dfe1bada6858a2aeb46a8f40606dd5403a27e98e